README.md

```diff
@@ -1 +1,32 @@
-# Example Custom operators and templates to be used with the Alpine Data Science Platform.
+# Alpine Custom Operator Framework
+
+Customers can now create Custom Operators to integrate with Alpine. This opens up a new world of possibilities for data scientists and engineers to expand Alpine's functionality. 
+
+- This framework allows data scientists and programmers to create a wide variety of Custom Operators to be used within an Alpine analytics workflow.
+- The Custom Operators can consume one or more tabular datasets and may produce a tabular dataset as output to following operators. 
+- Custom Operators can be written in either Java or Scala.
+- The Custom Operator framework supports Spark and Database operators. 
+- Once Custom Operators are deployed to an Alpine instance, the end user will not need to read, write or understand any code in order to use the new Custom Operator.
+- Specifically, users can easily integrate with their own specific Spark jobs or run their own custom code against their data within the Alpine framework.  
+
+As a result, an entire customer team can share pre-existing libraries of Analytics and have those incorporated into the Alpine visual workflows as simply another operator that can be chosen and configured through the Alpine drag and drop interface very quickly. 
+
+## What Version Should I Use?
+
+There are different versions of the Custom Operator Framework that map to different versions of the Alpine software. For more information, see the page here: [How to Update Custom Operators With New SDK Changes](https://alpine.atlassian.net/wiki/display/KB/How+to+Update+Custom+Operators+With+New+SDK+Changes)
+
+Alpine Version | Custom Operator SDK Version | Spark Version
+-------------- | --------------------------- | -------------
+5.6.1          | 1.0                         | 1.3.1
+5.7.1          | 1.3                         | 1.5.1
+5.8            | 1.4.2                       | 1.5.1
+
```

Comment:
It probably fine. Eventually it will change but we will just deal with that
when we get there

On Wed, Jan 13, 2016 at 4:10 PM, nelsonam <notifications@github.com> wrote:

> In README.md
> <https://github.com/AlpineNow/CustomPlugins/pull/10#discussion_r49671474>:
>
> > +- The Custom Operator framework supports Spark and Database operators.
> > +- Once Custom Operators are deployed to an Alpine instance, the end user will not need to read, write or understand any code in order to use the new Custom Operator.
> > +- Specifically, users can easily integrate with their own specific Spark jobs or run their own custom code against their data within the Alpine framework.
> > +
> > +As a result, an entire customer team can share pre-existing libraries of Analytics and have those incorporated into the Alpine visual workflows as simply another operator that can be chosen and configured through the Alpine drag and drop interface very quickly.
> > +
> > +## What Version Should I Use?
> > +
> > +There are different versions of the Custom Operator Framework that map to different versions of the Alpine software. For more information, see the page here: [How to Update Custom Operators With New SDK Changes](https://alpine.atlassian.net/wiki/display/KB/How+to+Update+Custom+Operators+With+New+SDK+Changes)
> > +
> > +Alpine Version | Custom Operator SDK Version | Spark Version
> > +-------------- | --------------------------- | -------------
> > +5.6.1          | 1.0                         | 1.3.1
> > +5.7.1          | 1.3                         | 1.5.1
> > +5.8            | 1.4.2                       | 1.5.1
> > +
>
> Do you think it's necessary Rachel? Do they change per version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AlpineNow/CustomPlugins/pull/10/files#r49671474>.
>